rustls-cert-gen/src/main.rs

```diff
@@ -36,3 +36,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 	fs::write("certs/key.der", &cert.serialize_private_key_der())?;
 	Ok(())
 }
+
+#[test]
```

Comment:
This probably shouldn't get added?

Reply:
Its only there to demonstrate that tests are being run on both crates. I can remove it now, but that whole file is going to get replaced in the next PR anyway.